site/docs/master/README.md

```diff
@@ -36,19 +36,7 @@ If you encounter issues, review the [troubleshooting docs][30], [file an issue][
 
 ## Contributing
 
-Thanks for taking the time to join our community and start contributing!
-
-Feedback and discussion are available on [the mailing list][24].
-
-### Before you start
-
-* Please familiarize yourself with the [Code of Conduct][8] before contributing.
-* See [CONTRIBUTING.md][5] for instructions on the developer certificate of origin that we require.
-* Read how [we're using ZenHub][26] for project and roadmap planning
-
-### Pull requests
-
-* We welcome pull requests. Feel free to dig through the [issues][4] and jump in.
+If you are ready to jump in and test, add code, or help with documentation, follow the instructions on our [Start contributing](https://velero.io/docs/master/start-contributing/) documentation for guidance on how to setup Velero for development.
```

Comment:
We probably want to make this change to the root repo `README.md` as well since this is just a copy of it